src/screens/Asl/components/AslTable/index.js

```diff
@@ -106,10 +106,6 @@ const AslTable = props => {
       return 'Example'
     }
 
-    // const getVideoLink = (source, uniqueASLIdentifier) => {
-      
-    // }
-
     const handleVideo = (source, uniqueASLIdentifier) => {
       if (source === 'ASLCORE') {
         setVideoUrl(`https://ct-dev.ncsa.illinois.edu/data/aslvideos/aslcore/original/${uniqueASLIdentifier}.mp4`);
```

Comment:
This wont work in production - or when the hostname changes.
https:///data/aslvideos/aslcore/original/${uniqueASLIdentifier}.mp4
might work. But ideally you get this from the server.

Reply:
(we can't hard code server hosts names)